roles/kube_vip/templates/kube-vip-bgp.yaml.j2

```diff
@@ -0,0 +1,63 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  creationTimestamp: null
+  name: kube-vip
+  namespace: kube-system
+spec:
+  containers:
+  - args:
+    - manager
+    env:
+    - name: vip_arp
+      value: "false"
+    - name: port
+      value: "6443"
+    - name: vip_interface
+      value: "{{ kube_vip_interface }}"
+    - name: vip_cidr
+      value: "32"
+    - name: cp_enable
+      value: "true"
+    - name: cp_namespace
+      value: kube-system
+    - name: vip_ddns
+      value: "false"
+    - name: bgp_enable
+      value: "true"
+    - name: bgp_routerid
+      value: "{{ kube_vip_bgp_routerid }}"
+    - name: bgp_as
+      value: "{{ kube_vip_bgp_as }}"
+    - name: bgp_peeraddress
+    - name: bgp_peerpass
```

Comment:
Could you possibly add those as options too?  And if you can make them there if there is a value defined.

Reply:
made change